### ISSUE TITLE
Accessibility: The BoundingRectangle is incorrect when clicking the list & list item in inspect tree

### ISSUE THREAD

**Issue by Amy-Li03** [dotnet/winforms#3655]
* .NET Core Version: 
.NET SDK 5.0.100-rc.1.20377.3

* More Info: 
This is not a regression issue, it also reproduce on .NET Core 3.1.

* Have you experienced this same bug with .NET Framework?: 
Yes

**Problem description:**
The BoundingRectangle is incorrect, the yellow rectangle has gone out the boundary of form when clicking the "Standard" list below "FlatStyle" tree item in inspect tree.
![Core](https://user-images.githubusercontent.com/62929087/88616361-90d1ec00-d0c6-11ea-86fe-5be7fc2427e8.png)

**Expected behavior:**
 The yellow rectangle should surround the FlatStyle property.

**Minimal repro:**
1. Launch Inspect tool.
2. Create WinForms app with PropertyGrid and Button control.
3. Associate the Button with PropertyGrid as a selected object.
4. Run the app.
5. Click the "Flat" list item of FlatStyle property in inspect tree, then observe the BoundingRectangle.

**Comment by vladimir-krestov** [comment]
In my opinion, this is not a bug. This is issue of the incorrect test case. Inspect stores outdated information about the drop-down ListBox, because it has been already collapsed. So the ListBox is not available for the accessibility tree, if you update "FlatStyle" tree item (click "-", then "+" of the tree item), you will see, that it's true - there is no ListBox in the tree. The same for the ListBox items, they are available until the tree node updating.
Narrator works well, because it updates the tree. So, I would prefer just to close this bug.
cc/ @RussKie, @merriemcgaw

**Comment by RussKie** [comment]
I had a chat with @vladimir-krestov offline, and it looks like this is an Inspect-specific issue. Inspect is a developer specific tooling, and it's unlikely to be used by end users with accessibility needs.

@Amy-Li03 can you check if this issue affects end-user accessibility tools, such as Narrator, NVDA, etc. If those tools don't have the same problem - please close it.

**Comment by Amy-Li03** [comment]
@RussKie Checked this issue with accessibility tool: accessibility Insights, it doesn't have the same problem. And For Narrator, NVDA tool, they doesn't exist tree, so we cannot verify this problem, but it can announce the correct information when focusing.
So close this issue.
